packages/components/popover/src/Popover.tsx

```diff
@@ -213,7 +213,7 @@ export function Popover(props: PopoverProps) {
             return;
           }
 
-          closeAndFocusTrigger();
+          onClose();
```

Comment:
```suggestion
          onClose?.();
```
`onClose` prop can be undefined.
I see that TS doesn't warn about this, it's because of a `Function` type in props `onClose?: Function`
Could you change it please to `() => void` 🙇‍♂️